scripts/fmt.sh

```diff
@@ -13,8 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.#
 
+# Fix: 'fatal: detected dubious ownership in repository',
+# only do this if it is running into the develop image
+if [ "${PWD}" == "/opt/app" ]; then
+  git config --global --add safe.directory "*"
```

Comment:
I think we should do this inside the Dockerfile since that is where it's needed, e.g. https://github.com/kserve/modelmesh-serving/blob/eb4bf296ff4667d064eb6d71362adf5427c8f3c3/Dockerfile.develop#L114-L115

Reply:
moved.